ui/raidboss/data/06-ew/raid/p1n.ts

```diff
@@ -0,0 +1,116 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFirstCircle,
+  timelineFile: 'p1n.txt',
+  triggers: [
+    {
+      id: 'P1N Gaoler\'s Flail Right',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6DA2', source: 'Erichthonios', capture: false }),
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'P1N Gaoler\'s Flail Left',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6DA3', source: 'Erichthonios', capture: false }),
+      response: Responses.goRight(),
+    },
+    /* Aetherflail sends startsUsing Gaoler's Flail as well so don't double tell
+    {
+      id: 'P1N Aetherflail Right',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65DF', source: 'Erichthonios', capture: false }),
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'P1N Aetherflail Left',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65E0', source: 'Erichthonios', capture: false }),
+      response: Responses.goRight(),
+    },
+    */
```

Comment:
I think no need for commented out trigger blocks.  I think if you wanted to leave a note on the Gaoler's Flail triggers above as a comment that `Also happens during Aetherflail Right (65DF)` or something, that would be a better way to leave a note.

```suggestion
```

Reply:
Sure, I changed it in 198dfa351cc0138c29a91ee5515c1580f4b6d9ae